terraform/implementation/setup/backend.tf

```diff
@@ -3,10 +3,20 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "=5.70.0"
+      version = "~> 5.56.1"
```

Comment:
Did you intentionally go down a version for hashicorp/aws?

Reply:
Yes, this was intentional! It was never run with 5.7.